Source/WebCore/platform/graphics/avfoundation/InbandTextTrackPrivateAVF.cpp

```diff
@@ -488,19 +488,89 @@ void InbandTextTrackPrivateAVF::resetCueValues()
 
 void InbandTextTrackPrivateAVF::setMode(InbandTextTrackPrivate::Mode newMode)
 {
-    if (!m_owner)
-        return;
-
     InbandTextTrackPrivate::Mode oldMode = mode();
     InbandTextTrackPrivate::setMode(newMode);
 
-    if (oldMode == newMode)
+    if (m_owner && oldMode != newMode)
+        m_owner->trackModeChanged();
+}
+
+bool InbandTextTrackPrivateAVF::processVTTFileHeader(CMFormatDescriptionRef formatDescription)
+{
+    using namespace PAL;
+    ASSERT(!m_haveReportedVTTHeader);
+    ASSERT(formatDescription);
+
+    RefPtr<ArrayBuffer> buffer;
+
+    CFDictionaryRef extensions = PAL::CMFormatDescriptionGetExtensions(formatDescription);
+    if (!extensions)
+        return false;
+
+    CFDictionaryRef sampleDescriptionExtensions = static_cast<CFDictionaryRef>(CFDictionaryGetValue(extensions, kCMFormatDescriptionExtension_SampleDescriptionExtensionAtoms));
```

Comment:
`auto`

Reply:
Oh yeah, there are several places where I moved old code around that can now use `auto`. Will update.